tools/mochaRunner/src/main.ts

```diff
@@ -129,14 +129,31 @@ async function main() {
         ? statsFilename
         : path.join(tmpDir, 'output.json');
       console.log('Running', JSON.stringify(parameters), tmpFilename);
+      const reporterArgumentIndex = process.argv.indexOf('--reporter');
       const args = [
         '-u',
         path.join(__dirname, 'interface.js'),
         '-R',
-        path.join(__dirname, 'reporter.js'),
+        reporterArgumentIndex === -1
```

Comment:
I believe mochaRunner won't work with any standard reporter. Do you plan to re-implement a compatible reporter?

Reply:
yes, we extend JSON reporter (you can check here: https://phabricator.services.mozilla.com/differential/changeset/?ref=5931554), and it seems to be working